src/EdgeAnalysis.jl

```diff
@@ -457,11 +457,11 @@ function calcbranchingnumbers(sim,reactantinds,productinds,corespcsinds,corerxni
         index = edgerxninds[ind]
         reactionrate = edgereactionrates[ind]
         if reactionrate > 0
-            reactantside = reactantinds[:,ind]
-            productside = productinds[:,ind]
+            @views reactantside = reactantinds[:,index]
```

Comment:
Can you squash this to the relevant commit

Reply:
done